gson/bnd.bnd

```diff
@@ -17,3 +17,9 @@ Import-Package: sun.misc;resolution:=optional, *
     com.google.gson.annotations,\
     com.google.gson.reflect,\
     com.google.gson.stream
+
+# Ignore error about module-info.class being in wrong directory
+# See related https://issues.apache.org/jira/browse/FELIX-5592 (but that is for different plugin)
+# and https://github.com/bndtools/bnd/issues/2227
+-fixupmessages:\
+    "Classes found in the wrong directory: \\{META-INF/versions/9/module-info\\.class=module-info\\}";is:=warning
```

Comment:
If I understand https://github.com/bndtools/bnd/issues/2227 correctly, OSGi won't be able to load classes from `META-INF/versions`. But for `module-info.class` this might not matter because the OSGi manifest specifies the required packages.
The only issue might be packages which are since Java 9 part of a different optional module (e.g. `java.sql`), however for them maybe we should adjust `bnd.bnd` and specify these packages (in case that ever comes up as issue).